apps/voting/contracts/Voting.sol

```diff
@@ -47,6 +52,8 @@ contract Voting is IForwarder, AragonApp {
         uint256 votingPower;
         bytes executionScript;
         mapping (address => VoterState) voters;
+        mapping (address => address) casters;
```

Comment:
I think it would still be nice to add a bit of comments explaining how `casters` is used (only for representatives casting votes).